pkg/tfvars/openstack/rhcos_image.go

```diff
@@ -110,6 +73,52 @@ func uploadBaseImage(cloud string, localFilePath string, imageName string, clust
 	return nil
 }
 
+func importImage(imageName, imageID string, conn *gophercloud.ServiceClient, f *os.File) error {
+	logrus.Debugf("Using Image Import API to upload RHCOS to the image %q with ID %q", imageName, imageID)
```

Comment:
nit, no need to say RHCOS , this is a function to upload an image. In the future it could be used by something else besides trying to upload and RHCOS image.